src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -190,11 +190,14 @@ public sealed partial class TokenCache : ITokenCacheInternal
                             identityLogger: requestParams.RequestContext.Logger.IdentityLogger,
                             piiLoggingEnabled: requestParams.RequestContext.Logger.PiiLoggingEnabled);
 
-                        Stopwatch sw = Stopwatch.StartNew();
+                        var measuredResultDuration = await Task.Run( async () => 
```

Comment:
Not sure if this is the best way to take advantage of the MISE implementation. It is really nice but not super versatile. thoughts?
@bgavrilMS @pmaytak